scheduler-interface-spec.md

```diff
@@ -607,12 +607,10 @@ message NodeInfo {
   // node or schedule more allocations accordingly.
   Resource schedulableResource = 4;
 
-  // when the scheduler is co-exist with some other schedulers, some node
-  // resources might be occupied (allocated) by other schedulers.
-  Resource occupiedResource = 5;
-
   reserved 6;
   reserved "existingAllocations";
+  reserved 5;
```

Comment:
I think we should split this PR into two phases -- one which adds the new constants, and a second which removes the old API (to be committed after all changes have been made in both core and shim). Tracking non-YK pods is a huge feature, and this gives us more flexibility in the interim.